lib/parser.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+// Parses protocol data unit id
+//   buffer - <Buffer>, source buffer
+//   offset - <number>, number of bytes to skip before starting to read
+// Returns: <number>, id of protocol data unit
+const parseDataUnitId = (buffer, offset) => buffer.readUInt8(offset);
+
+// Parses handshake from buffer
+//   buffer - <Buffer>, source buffer
+// Returns: <Object>, handshake
+const parseHandshake = buffer => ({
+  version: buffer.readUInt16LE(0),
```

Comment:
For now, I think it's okay to use magic numbers here. But we'll need functions for writing to the buffer using the same offsets.
So it's better to make them constants or even considering making a data schema like
```
schema = [{ name: 'version', size: 4}, { name: 'status', size: 2}, { name: 'token', size: ]
```